src/components/Checkout/OrderSummary.js

```diff
@@ -1,163 +1,179 @@
+    }
 
-		stripe.retrievePaymentIntent(clientSecret).then(({ paymentIntent }) => {
-			switch (paymentIntent.status) {
-				case 'succeeded':
-					setMessage('Payment succeeded!');
-					break;
-				case 'processing':
-					setMessage('Your payment is processing.');
-					break;
-				case 'requires_payment_method':
-					setMessage('Your payment was not successful, please try again.');
-					break;
-				default:
-					setMessage('Something went wrong.');
-					break;
-			}
-		});
-	}, [stripe]);
+    stripe.retrievePaymentIntent(clientSecret).then(({ paymentIntent }) => {
+      switch (paymentIntent.status) {
+        case "succeeded":
+          setMessage("Payment succeeded!");
+          break;
+        case "processing":
+          setMessage("Your payment is processing.");
+          break;
+        case "requires_payment_method":
+          setMessage("Your payment was not successful, please try again.");
+          break;
+        default:
+          setMessage("Something went wrong.");
+          break;
+      }
+    });
+  }, [stripe]);
 
-	return (
-		<VStack
-			w='full'
-			p={10}
-			spacing={10}
-			alignItems='flex-start'
-			style={{
-				border: '2px solid #E2E8F0',
-				borderRadius: '25px',
-			}}
-		>
-			<Heading size='xl'>Order Summary</Heading>
-			<SimpleGrid columns={2} rowGap={6} columnGap={2} w='full'>
-				<GridItem colSpan={1}>
-					<Text>{`Items(00):`}</Text>
-				</GridItem>
-				<GridItem colSpan={1}>
-					<Text>{`$ 000.00`}</Text>
-				</GridItem>
-				<GridItem colSpan={1}>
-					<Text>{`Shipping and handling:`}</Text>
-				</GridItem>
-				<GridItem colSpan={1}>
-					<Text>{`$ 00.00`}</Text>
-				</GridItem>
-				<GridItem colSpan={1}>
-					<Text>{`Tax:`}</Text>
-				</GridItem>
-				<GridItem colSpan={1}>
-					<Text>{`$ 00.00`}</Text>
-				</GridItem>
-				<GridItem colSpan={2}>
-					<Divider />
-				</GridItem>
-				<GridItem colSpan={1}>
-					<Heading size='l'>Order total:</Heading>
-				</GridItem>
-				<GridItem colSpan={1}>
-					<Heading size='l'>{`$ 00.00`}</Heading>
-				</GridItem>
-				<GridItem colSpan={2}>
-					<Button
-						disabled={isLoading || !stripe || !elements}
-						size='md'
-						w='full'
-						bg={colors.primary}
-						color='white'
-						onClick={handleSubmit}
-					>
-						<span id='button-text'>
-							{isLoading ? (
-								<div className='spinner' id='spinner'></div>
-							) : (
-								'Place Order'
-							)}
-						</span>
-					</Button>
-				</GridItem>
-				<GridItem colSpan={2}>
-					{message && (
-						<div
-							style={{
-								color: 'rgb(105, 115, 134)',
-								fontSize: '16px',
-								lineHeight: '20px',
-								paddingTop: '12px',
-								textAlign: 'center',
-							}}
-						>
-							{message}
-						</div>
-					)}
-				</GridItem>
-			</SimpleGrid>
-		</VStack>
-	);
+  return (
+    <VStack
+      w='full'
+      p={10}
+      spacing={10}
+      alignItems='flex-start'
+      style={{
+        border: "2px solid #E2E8F0",
```

Comment:
You can use Chakra's border and borderRadius props to do this instead of overriding styles.